### DISCUSSION TITLE
MLFlow UI and save_dir specification

### DISCUSSION THREAD

**Discussion by ghost** [Lightning-AI/pytorch-lightning#16112] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
Hello,

When I use the MLFlowLogger without setting `save_dir` argument, then run `mlflow ui`, I have no issue to display artefacts.
In `mlruns/438809872721685038/d0b0373f48c549f3b36e97f5017dda71/meta.yml,` I see
`artifact_uri: file:./mlruns/438809872721685038/d0b0373f48c549f3b36e97f5017dda71/artifacts`

When I set `save_dir` (here "experiments/mlruns"), then run `cd experiments;mlflow ui`, I don't see the artefacts.
In `experiments/mlruns/933121669842833043/64cd5a18fe71423d8c9423a320ee7b91/meta.yml`, I see
`artifact_uri: file:experiments/mlruns/933121669842833043/64cd5a18fe71423d8c9423a320ee7b91/artifacts`

I fix it by setting the `tracking_uri` to set an absolute path, like this :
```
save_dir = "experiments/mlruns"
pl.loggers.mlflow.MLFlowLogger(
    save_dir=save_dir,
    experiment_name=experiment_name,
    run_name=run_name,
    tracking_uri=f"file://{save_dir}"
)
```

Is there a better way to fix it ?
Is it a good idea to open an issue about it ?

**Comment by akihironitta** [comment, marked as answer]
@NicolasNeovision It would be great if you could submit a bug report by creating a GitHub issue with a minimal script and env detail. Thank you ⚡

**Reply by ghost** [reply]
Thanks for the answer.

I opened #16205
